### PR TITLE
Add comments to the smart contract examples

### PR DESCRIPTION
**Pull request by moCello**
Resolves: #130

### CODE REVIEW

**Review by ureeves — COMMENTED** [review]
Some comments 😊

File: modules/eventer/src/lib.rs [hunk 1]
```diff
@@ -6,7 +6,6 @@
 
 #![feature(arbitrary_self_types)]
 #![no_std]
-#![no_main]
```
[the remark this answers is not in the document]
Comment (moCello) [reply modules/eventer/src/lib.rs:9]: it is, this commit should disappear from this pr once [this PR](https://github.com/dusk-network/piecrust/pull/128) is merged

File: modules/everest/src/lib.rs [hunk 2]
```diff
@@ -4,28 +4,34 @@
 //
 // Copyright (c) DUSK NETWORK. All rights reserved.
 
+//! Module to get the current block-height from the host.
+
 #![feature(arbitrary_self_types)]
 #![no_std]
-#![no_main]
```
[the remark this answers is not in the document]
Comment (moCello) [reply modules/everest/src/lib.rs:9]: ditto

File: modules/fallible_counter/counter/src/lib.rs [hunk 3]
```diff
@@ -4,40 +4,49 @@
 //
 // Copyright (c) DUSK NETWORK. All rights reserved.
 
+//! Module to implement a simple counter that can be read and incremented by
+//! one count.
+
 #![feature(arbitrary_self_types)]
 #![no_std]
-#![no_main]
```
[the remark this answers is not in the document]
Comment (moCello) [reply modules/fallible_counter/counter/src/lib.rs:9]: ditto

File: modules/fibonacci/src/lib.rs [hunk 4]
```diff
@@ -4,23 +4,28 @@
 //
 // Copyright (c) DUSK NETWORK. All rights reserved.
 
+//! Module that calculates the nth number of the Fibonacci sequence
+
 #![feature(arbitrary_self_types)]
 #![no_std]
-#![no_main]
```
[the remark this answers is not in the document]
Comment (moCello) [reply modules/fibonacci/src/lib.rs:9]: ditto :P

File: modules/fallible_counter/counter/src/lib.rs [hunk 5]
```diff
@@ -4,40 +4,49 @@
 //
 // Copyright (c) DUSK NETWORK. All rights reserved.
 
+//! Module to implement a simple counter that can be read and incremented by
+//! one count.
```
[the remark this answers is not in the document]
Comment (moCello) [reply modules/fallible_counter/counter/src/lib.rs:8]: I also seems as if we don't even call the nested version of `counter` from `fallible_counter`. I'll remove it

**Pull request closed without merging**